### DISCUSSION TITLE
Haked? How to know

### DISCUSSION THREAD

**Discussion by nacarrillos** [community/community#36376] [Code Security]
Hello everyone, I have a repository on GitHub and since some weeks there is a "Collaborator" that somehow GitHub says is editing my code. There is nonetheless no Commit from his/her side, i never gave this User access and the worst is that when one click on this user, gITHUB says "404 Error". The description from the user says "hacked"...Do anyone know what  could mean and if I should be worried?
![Captura de pantalla 2022-10-17 095247](https://user-images.githubusercontent.com/104648692/196120286-dcd58de7-a082-418d-be9f-316386deeedf.png)
![Captura de pantalla 2022-10-17 095257](https://user-images.githubusercontent.com/104648692/196120336-c9100d4b-9bd6-4d60-891b-432b3694bd36.png)

**Comment by ossanna16** [comment]
Hey there @nacarrillos and thank you for asking a good question! I'm glad you found our community 🙂
